lib/impure/nre.nim

```diff
@@ -432,7 +432,12 @@ proc initRegex(pattern: string, flags: int, study = true): Regex =
 
   if study:
     # XXX investigate JIT
```

Comment:
It appears to me as if JIT has been investigated and this line can be safely removed :grin:.